docs/scos/dev/back-end-development/project-namespaces.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
we are missing some required information here

We need describe how we can configure namespaces. What hierarchy and inheritance we are using with them.
Also we need to provide use cases for what we need this functionality and how we can use it.

Aslo we have a few development tool that uses that configurable namespaces and we need to mention them too.

Looks like Metro project used it, so you might ask them for examples.